extension-compose/src/main/java/com/mapbox/maps/extension/compose/MapEffect.kt

```diff
@@ -0,0 +1,100 @@
+package com.mapbox.maps.extension.compose
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.currentComposer
+import com.mapbox.maps.MapView
+import com.mapbox.maps.MapboxExperimental
+import com.mapbox.maps.extension.compose.internal.MapApplier
+import kotlinx.coroutines.CoroutineScope
+
+/**
+ * When MapEffect enters the composition it will launch [block] into the composition's CoroutineContext.
+ * The coroutine will be cancelled and re-launched when [MapEffect] is recomposed with a different [key1].
+ *
+ * The coroutine will be cancelled when the [MapEffect] leaves the composition.
+ *
+ * This function should not be used to (re-)launch ongoing tasks in response to callback events by
+ * way of storing callback data in MutableState passed to key. Instead, see rememberCoroutineScope
```

Comment:
nit: maybe a link to `see rememberCoroutineScope`?

Reply:
rememberCoroutineScope is well defined in the coroutine package and is not used in this file, thus links won't be resolved properly, will leave it as is.